test/e2e/json-rpc/eth_gasPrice.spec.js

```diff
@@ -0,0 +1,46 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('eth_gasPrice', function () {
+  const ganacheOptions = {
```

Comment:
same as above

Reply:
Done in [3f4323d](https://github.com/MetaMask/metamask-extension/pull/20963/commits/3f4323d5abc0b8c1993d5bcf98015f12a5405d5e)